hack/do-janitor/do-janitor.go

```diff
@@ -85,6 +85,25 @@ func main() {
 		}
 	}
 
+	volumes, err := volumeList(ctx, client)
+	if err != nil {
+		log.Fatalf("failed to list volumes: %+v", err.Error())
+	}
+
+	for _, volume := range volumes {
+		hours := time.Since(volume.CreatedAt).Hours()
+		if hours >= timeToCleanInHours {
+			log.Printf("%s is older than %d hours will terminate\n", volume.Name, timeToCleanInHours)
+			_, err := client.Storage.DeleteVolume(ctx, volume.ID)
```

Comment:
I think the `DeleteVolume` call would fail if a droplet is still attached to it. That should not fail in the general case since we first delete the droplets, but we might still run into it and produce scary-looking error messages in edge cases and/or when droplets and volume deletes race.

In addition to filtering for the create timestamp, if we also check the [`DropletIDs`](https://github.com/digitalocean/godo/blob/de394a0f64bde5a3bb860c1b6547a098b50aa2aa/storage.go#L52) and `continue` the loop if it's non-empty (meaning that the volume is still attached to a droplet), we can avoid this problem.

Reply:
Sure, updated. Please let me know if this looks okay now.